docroot/themes/humsci/su_humsci_theme/color/preview.js

```diff
@@ -0,0 +1,50 @@
+
+(function ($, Drupal, drupalSettings) {
+  Drupal.color = {
+    logoChanged: false,
+    callback: function callback(context, settings, $form) {
+      if (drupalSettings.color.userId == 1) {
+        $('.color-placeholder, .color-palette').show();
+      }
+      if (!this.logoChanged) {
+        $('.color-preview .color-preview-header img').attr('src', drupalSettings.color.logo);
+        $('.color-preview .color-preview-global-footer img').attr('src', drupalSettings.color.globalFooterLogo);
+        this.logoChanged = true;
+      }
+
+      if (drupalSettings.color.logo === null) {
+        $('div').remove('.color-preview-logo');
+      }
+
+      var $colorPreview = $form.find('.color-preview');
+      var $colorPalette = $form.find('.js-color-palette');
+
+
+      $colorPreview.find('.color-preview-brand-bar').css('background-color', $colorPalette.find('input[name="palette[branding]"]').val());
+      $colorPreview.find('.color-preview-brand-bar').css('color', $colorPalette.find('input[name="palette[brandingcolor]"]').val());
+
+      $colorPreview.find('.color-preview-header').css('background-color', $colorPalette.find('input[name="palette[header]"]').val());
+      $colorPreview.find('.color-preview-header').css('color', $colorPalette.find('input[name="palette[headertext]"]').val());
+      $colorPreview.find('.color-preview-header a').css('color', $colorPalette.find('input[name="palette[headerlink]"]').val());
+
+      $colorPreview.find('.color-preview-main').css('background-color', $colorPalette.find('input[name="palette[main]"]').val());
+      $colorPreview.find('.color-preview-main').css('color', $colorPalette.find('input[name="palette[maintext]"]').val());
+      $colorPreview.find('.color-preview-main a').css('color', $colorPalette.find('input[name="palette[mainlink]"]').val());
+
+      $colorPreview.find('.color-preview-footer').css('background-color', $colorPalette.find('input[name="palette[footer]"]').val());
+      $colorPreview.find('.color-preview-footer').css('color', $colorPalette.find('input[name="palette[footertext]"]').val());
+      $colorPreview.find('.color-preview-footer a').css('color', $colorPalette.find('input[name="palette[footerlink]"]').val());
+
+      $colorPreview.find('.color-preview-global-footer').css('background-color', $colorPalette.find('input[name="palette[globalfooter]"]').val());
+      $colorPreview.find('.color-preview-global-footer, .color-preview-global-footer a').css('color', $colorPalette.find('input[name="palette[globalfooterlink]"]').val());
+
+      $colorPreview.find('.color-preview-decanter-button').css('background-color', $colorPalette.find('input[name="palette[button]"]').val());
+      $colorPreview.find('.color-preview-decanter-button').css('color', $colorPalette.find('input[name="palette[buttoncolor]"]').val());
+
+      // var gradientStart = $colorPalette.find('input[name="palette[top]"]').val();
```

Comment:
Why commented out code?

Reply:
This file was adaptbed from bartik theme. And I left these here in case we want to try them out for something.

Reply:
Ok, if the code is not intended to be fixed or used and is only a "maybe in the future" we should just remove it. Especially if the code sample can be found elsewhere in the project.